packages/cli/src/commands/config/set.ts

```diff
@@ -10,19 +10,28 @@ export default class Set extends LocalCommand {
     // Overrides base command node flag.
     node: flags.string({
       char: 'n',
-      required: true,
       description: 'URL of the node to run commands against',
-      default: 'http://localhost:8545',
```

Comment:
Mmmm, if you decide to remove the defaults from the commands (for node and gasCurrency) but you are adding those from the config utils, you should at least add in the descriptions that those are the defaults.

It's kind of weird having the default here and in the configFile and it seems wrong, I know, but I think that this default helps a lot for the documentation.